dockers/docker-teamd/Dockerfile.j2

```diff
@@ -26,5 +26,4 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
 RUN rm -rf /debs
 
-ENTRYPOINT ["/bin/bash", "-c"]
-CMD ["/usr/bin/config.sh && /usr/bin/start.sh"]
+ENTRYPOINT ["/usr/bin/start.sh"]
```

Comment:
I think we could use /bin/bash as ENTRYPOINT and /usr/bin/start.sh as CMD. Thus even when the start.sh fails at some unknown point, the docker won't exit and dev could attach to the docker and debug. The Dockerfile could be similar to docker-orchagent/Dockerfile.j2.